modules/sap_hana_svpc/variables.tf

```
@@ -14,38 +14,36 @@
  * limitations under the License.
  */
 
-variable "primary_instance_name" {
-  description = "A unique name for the resource, required by GCE. Changing this forces a new resource to be created."
-}
-
-variable "secondary_instance_name" {
-  description = "A unique name for the resource, required by GCE. Changing this forces a new resource to be created."
+variable "project_id" {
+  description = "The ID of the service project in which the resources will be deployed."
 }
 
-variable "primary_zone" {
-  description = "The primary zone that the instance should be created in."
+variable "subnetwork_project" {
+  description = "The ID of the shared VPC host project in which the service project's will be deployed."
```

The description should be clarified for all instances of this variable.

```suggestion
  description = "The ID of the shared VPC host project which contains the subnetwork."
```